UM/Math/Color.py

```diff
@@ -67,3 +67,25 @@ def fromARGB(value):
             (value & 0x000000ff) >> 0,
             (value & 0xff000000) >> 24
         )
+
+    ##  Returns a new Color constructed from a 7-character string "#RRGGBB" format.
+    #
+    #   \param value A 7-character string representing a color in "#RRGGBB" format.
+    #   \return A Color constructed from the components of value.
+    @staticmethod
+    def fromRGBString(value):
```

Comment:
It would be nice if you could turn this into a "fromRGBAString" method that detects whether the string is RGB or RGBA. That way we do not have to add an extra function for that later.